vite.config.js

```diff
@@ -14,15 +14,15 @@ export default defineConfig({
     reactPlugin({
       // Only run fast-refresh for src
       fastRefresh: true,
-      include: 'src/**',
+      include: 'src',
 
       // Add extra emotion engine features
       jsxImportSource: '@emotion/react',
     }),
 
     // Compress everything with Brotli while building, so we don't have to do it on the fly
     compressionPlugin({
-      algorithm: 'brotliCompress',
+      algorithm: 'brotli',
       filter: /\.(js|jsx|ts|tsx|css|svg|ico)$/i,
       verbose: false,
     }),
```

Comment:
**Suggestion:** Correct the algorithm name for the compressionPlugin. It currently is 'brotli', but it should be 'brotliCompress'.
```suggestion
      compressionPlugin({
        algorithm: 'brotliCompress',
        filter: /\.(js|jsx|ts|tsx|css|svg|ico)$/i,
        verbose: false,
      }),
```